extensions/parquet/base/src/main/java/io/deephaven/parquet/base/ParquetFileReader.java

```diff
@@ -37,6 +42,76 @@ public class ParquetFileReader {
     private final URI rootURI;
     private final MessageType type;
 
+    /**
```

Comment:
These methods have been moved from ParquetTools. They were called create getParquetFileReader and getParquetFileReaderChecked.

A few changes that I made:
- Renaming them to create and createChecked
- Instead of throwing a TableDataException, I am throwing UncheckedDeephavenException, because this package cannot access TableDataException.
- Instead of accepting ParquetInstructions, the create methods now accept specialInstructions which have type Object. This is again because this package cannot access ParquetInstructions. Also, updated the javadoc for this parameter.